public/app/core/services/util_srv.ts

```diff
@@ -52,18 +52,14 @@ export class UtilSrv {
   showConfirmModal(payload) {
     const scope = this.$rootScope.$new();
 
-    scope.onConfirm = () => {
```

Comment:
Breaking change? Can't find any code using confirm modal without `onConfirm`, but maybe breaking for some external plugin?

Reply:
You're right. Overwritten further down.